PyInstaller/_shared_with_waf.py

```diff
@@ -64,8 +64,10 @@ def _pyi_machine(machine, system):
     if machine.startswith(("ppc", "powerpc")):
         # PowerPC comes in 64 vs 32 bit and little vs big endian variants.
         return "ppc"
+    if machine in ("mips64", "mips"):
+        return "mips"
     # Machines with no known aliases :)
-    if machine in ("s390x", "mips"):
+    else:
```

Comment:
Surely you mean?
```suggestion
    if machine in ("s390x",):
```
Otherwise, you'd make `machine` the default which we definitely don't want or waf will be unable to put the bootloaders where PyInstaller expects to find them.

Reply:
Maybe for these cases that do not need to be processed, we only need to return `machine` and do not need to list a long list?

Reply:
Yes, the cases which we know have no aliases (or rather `platform.machine() == ctx.env.DEST_CPU`) can safely be returned as-is (by returning `machine`) but for unknown architectures that we've never seen before, it's unlikely that `platform.machine()` and `ctx.env.DEST_CPU` agree with each other so we must normalise to a fixed value (`"unknown"`).